### DISCUSSION TITLE
feature for config voices to show

### DISCUSSION THREAD

**Discussion by hd-i** [CoderLine/alphaTab#816] [Ideas]
Can you add a feature to config alphatab to select a voice with voice number(1,2,3,4) or show multivoice such as Guitarpro ?

**Comment by Danielku15** [comment]
Generally alphaTab supports rendering of multiple voices. As of today you can specify what the "voice 1" and alternate voices should have as a color. (https://alphatab.net/docs/reference/settings/display/resources#colors). Fine grain control about the colors of each voice are not available and could be an extension. 

Getting a switcher like in GP would mean: Change the settings and re-render.
